runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldHopperPlugin.java

```diff
@@ -161,6 +163,10 @@
 
 	private final Map<Integer, Integer> storedPings = new HashMap<>();
 
+	private List<World> worldCycleList = new ArrayList<>();
+
+	private final String NUMBER_COMMA_ONLY_REGEX = "((?<!^,)\\d+(,(?!$)|$))+";;
```

Comment:
Rather than using a new regex for splitting commas and checking numbers, use `Text.fromCSV` and filter any entries which aren't numbers.